src/submission/submission-to-clinical/stat-calculator.ts

```diff
@@ -64,8 +64,10 @@ const coreClinialSchemaNamesSet = new Set<CoreClinicalSchemaName>(
   Object.keys(schemaNameToCoreCompletenessStat) as CoreClinicalSchemaName[],
 );
 
+const dnaSampleTypes = ['Amplified DNA', 'ctDNA', 'Other DNA enrichments', 'Total DNA'];
```

Comment:
Where did this list come from? Is it likely to be expanded and maintained further? It likely should move to a more general constants file.

Reply:
I've moved dnaSampleTypes to the clinical-entities common definitions. Not really finding a better method for obtaining these values so we'll keep it as a string array for now.